src/test/util/setup_common.cpp

```diff
@@ -215,13 +215,13 @@ CTxMemPoolEntry TestMemPoolEntryHelper::FromTx(const CTransactionRef& tx)
 }
 
 /**
- * @returns a real block (0000000000013b8ab2cd513b0261a14096412195a72a0c4827d229dcc7e0f7af)
- *      with 9 txs.
+ * @returns a real block (0000000000000a482a11624608cfd526e34036ca80707ee2a2b610c8b3e362a7)
+ *      with 2 txs.
  */
-CBlock getBlock13b8a()
+CBlock getBlock6548()
```

Comment:
Took the block at least with 2 transactions. Now we have block with much more, probably can switch to one of those later.